src/Plugins/RunawayProcessKiller/RunawayProcessKillerExtension.cs

```diff
@@ -38,9 +44,12 @@ public RunawayProcessKillerExtension()
             // Default initializer
         }
 
-        public RunawayProcessKillerExtension(String pidfile)
+        public RunawayProcessKillerExtension(String pidfile, int stopTimeoutMs = 5000, bool stopParentFirst = false, bool checkWinSWEnvironmentVariable = true)
         {
             this.Pidfile = pidfile;
+            this.StopTimeout = TimeSpan.FromMilliseconds(5000);
```

Comment:
This is a bug, but this API is not being used anywhere outside tests. will fix in a follow-up